hedera-node/hedera-app/src/test/java/com/hedera/node/app/workflows/prehandle/PreHandleWorkflowImplTest.java

```diff
@@ -341,7 +342,8 @@ void preHandlePayerAccountNotFound() throws PreCheckException {
             // (Erin doesn't exist yet)
             final var txInfo = scenario().withPayer(FRANK.accountID()).txInfo();
 
-            final Transaction platformTx = new SwirldTransaction(asByteArray(txInfo.transaction()));
+            final Transaction platformTx = new PayloadWrapper(
+                    new OneOf<>(PayloadOneOfType.APPLICATION_PAYLOAD, Bytes.wrap(asByteArray(txInfo.transaction()))));
```

Comment:
I believe the last parameter can be replaced with `Transaction.PROTOBUF.toBytes(txInfo.transaction)`.